src/manifest.json

```diff
@@ -43,6 +43,12 @@
       },
       "description": "Open containers panel"
     },
+    "sort_tabs_by_container": {
+      "suggested_key": {
+        "default": "Ctrl+Comma"
```

Comment:
We should also add:
```
"mac": "MacCtrl+Comma"
```

Reply:
Added the suggested keyboard shortcut for mac.